cypress/test/navigation/DocsNav.cy.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { mount } from 'cypress/react';
+import DocsNav from '../../../components/navigation/DocsNav';
+import MockRouter from '../../utils/router';
+import { Docsitem } from '../../fixtures/navigation-mock';
+
+describe('DocsNav', () => {
+  it('renders the nav items with the correct styles and icons', () => {
+    const active = '/getting-started/installation';
+    mount(
+    <MockRouter> <DocsNav item={Docsitem} active={active}  /></MockRouter>
+   );
+
```

Comment:
Check for the active state and some of the Doc Items rendered inside the component.

Reply:
done

Reply:
How are you checking for active state of any item in DocNav? Can we do it using CSS properties?

Reply:
checking using changed class